config/initializers/date_formats.rb

```diff
@@ -1,21 +1,10 @@
-# The below hotfix is taken from https://stackoverflow.com/a/71328079
-# Overrides the rails 7 removal of to_s and allows our default
-# format to be applied rather than having to find each date
-# and manually add `.to_fs` to the end of it
+# The below hotfix overrides the rails 7 removal of to_s and
+# allows our default format to be applied rather than having
+# to find each date and manually add `.to_fs` to the end of it
```

Comment:
Is it worth adding a link to the release notes? I don't mind either way, but it might be useful for anyone curious.

https://guides.rubyonrails.org/7_0_release_notes.html#active-support-deprecations